src/coreclr/src/vm/eeconfig.h

```diff
@@ -916,6 +917,7 @@ class EEConfig
     size_t iGCgen0size;
     size_t iGCSegmentSize;
     int  iGCconcurrent;
+    int iGClarge;
```

Comment:
Do we need to cache it here? Can we just look this up directly in `GCToEEInterface::GetBooleanConfigValue` ?

Reply:
It is possible, but I would like to keep this consistent with the rest. As far as I know, all settings are cached, it would be weird to introduce inconsistency.

Reply:
Most of the caching of GC stuff that is going on here is useless. It would best to delete it, or at least stop the tread with adding more. It is leftover from the time when the GC was coupled with the runtime and did not have its own cache.